Dockerfile

```
@@ -2,32 +2,53 @@ ARG IMAGE=latest
 
 # Base image
 #============
-FROM renovate/yarn:1.22.4@sha256:b7c43fa2915bdc6082aae63260c0afe67ae18d013e6dbaf4b0d14c221dd4f0b7 AS base
+FROM renovate/buildpack AS base
 
 LABEL maintainer="Rhys Arkins <rhys@arkins.net>"
 LABEL name="renovate"
 LABEL org.opencontainers.image.source="https://github.com/renovatebot/renovate" \
-      org.opencontainers.image.url="https://renovatebot.com" \
-      org.opencontainers.image.licenses="AGPL-3.0-only"
+  org.opencontainers.image.url="https://renovatebot.com" \
+  org.opencontainers.image.licenses="AGPL-3.0-only"
 
 USER root
 WORKDIR /usr/src/app/
 
+# Python
+
+# renovate: datasource=docker depName=python
+ENV PYTHON_VERSION=3.8.2
+RUN install-tool python
```

```suggestion
RUN install-tool python 3.8.2
```

Is this possible now?

---

sure, should we do this for all tools?